lib/couchbase.js

```diff
@@ -15,12 +15,17 @@ var red   = chalk.red;
 module.exports = CoreObject.extend({
   init: function(options) {
     var self = this;
-    CoreObject.prototype.init(options);
+    var host = options.readConfig('host'),
+        bucketName = options.readConfig('bucketName');
 
-    if (!self.config) {
-      throw new SilentError("You have to pass a config!");
+    if (!host) {
+      throw new SilentError("You have to pass in a host");
     }
 
+    if (!bucketName) {
+      throw new SilentError("You have to pass in a bucketName");
+
+    self.config = {host: host, bucketName: bucketName};
```

Comment:
have you tested these changes? looks like there is no closing `}` for `if (!bucketName) {`.